testsuite/forge/src/backend/local/cargo.rs

```diff
@@ -136,7 +136,7 @@ where
     let output = Command::new("cargo")
         .current_dir(directory)
         .env("CARGO_TARGET_DIR", target_directory)
-        .args(&["build", "--bin=diem-node"])
+        .args(&["build", "--bin=diem-node", "--features=failpoints"])
```

Comment:
Does the mean for all the local forge tests, they will have failpoints on and trigger that use case?  That means they won't trigger the non-failpoints cases no?  It might be good to have tests for local forge that use failpoints and one that doesn't?

Reply:
cluster tests don't have failpoints enabled and runs 100 epoch test multiple times day so I'm not worried about happy path coverage.

Reply:
It's good to know that there is coverage in cluster test.  Still it's nice to have a local test that can be debugged upon a failure IMO and of course if cluster test was ever changed to use failpoints as well, then we might forget to test the happy path explicitly.

Reply:
maybe bootcamp task?